### PR TITLE
ED-1215 FAS redesign 1200

### PR DESCRIPTION
**Pull request by stevenburnell-dit**
Stupidly large PR for all the 1200 redesign changes

### CODE REVIEW

**Comment by richtier** [comment]
I think `/industries` is not as per design:

# design
image

# in PR
image

**Comment by richtier** [comment]
and company profile has a small difference:

# design
image

# the PR
image

(see the divider line has unexpected padding)

File: company/templates/company-contact-form.html [hunk 1]
```diff
@@ -11,43 +11,44 @@
...
-						<div class="span7" style="padding-left: 20px;">
-							<br>
+						<div class="span7">
 							<p>
 								Fill in your details and a brief message summarising your needs that will be sent to the UK company.
 							</p>
 							<br>
 							<div class="input-field-container">
-								{{ form.as_ul }}
+								{{ form.as_p }}
```
Comment (richtier) [inline company/templates/company-contact-form.html]: changing from `.as_ul` to `.as_p` breaks the error labels:

# as_ul
image


# as_p:
image

File: company/templates/company-search-results-list.html [hunk 2]
```diff
@@ -37,10 +36,10 @@
 					</div>
 
 					<div id="ed-search-filters-list">
-						<strong>Filter by industry</strong>
-						<div class="">
+						<fieldset>
+							<legend>Filter by industry</legend>
 							{{ form.sectors }}
```
Comment (richtier) [inline company/templates/company-search-results-list.html]: the highlight outline around the checkboxes are trimmed:

# the PR
image

# in dev
image

**Comment by richtier** [comment]
out checkboxes must be consistent with the styleguide.

# this pr:
image

# the styleguide
image

http://styleguide.uktrade.io/forms-inputs/

**Comment by richtier** [comment]
the "page in english only" box looks funky

image

**Comment by richtier** [comment]
lead generation form in has broken translation:

# in this pr

image


# in dev

image

**Comment by stevenburnell-dit** [comment]
> I think /industries is not as per design:
Correct. The original design doesn't match the change output. Callum changed his mind after I (spent a while implementing!) queried how he wanted the tablet/mobile views. They are now meant to be consistent with Home page.
https://uktrade.atlassian.net/browse/ED-1936?focusedCommentId=22401&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22401

 > and company profile has a small difference:
Yes, there are a few inconsistencies over margins and paddings, so I simply created some default CSS and let everything pull from that. Any intentional and highly desired inconsistencies can be added later once Callum has had a chance to review. I've made him of this, but will also send an email on this specific issue to get a definitive answer. Will CC you in.

> out checkboxes must be consistent with the styleguide.
The linked styleguide has not been created with FAS in mind (which has a different look). I've used the look from the designs created for this project, as there is no scope or requirement in ED-1215 to update designs+CSS to match that styleguide. 

> the "page in english only" box looks funky
Quite true, although I'm guessing you got the accurate issue description ('funky') from Jess :-)
Have tweaked to give better vertical alignment.

> lead generation form in has broken translation:
Correct. I have already raised a sub-task (https://uktrade.atlassian.net/browse/ED-1969) in the parent ticket. Needs attention from Rob. Will make sure I assign it to him.

> changing from .as_ul to .as_p breaks the error labels:
Well spotted. Turns out there was also inconsistency with error message placement between lead generation and customer contact forms. Have now correct both the CSS and placement for both forms. 
 
> the highlight outline around the checkboxes are trimmed:
Another 'well spotted'. Have now fixed.


Think that's everything. Let me know if I've overlooked any comment.

**Pull request merged**